api/REST-API.md

```diff
@@ -33,46 +33,49 @@ Today there are no projects or applications that don't have a REST API for the c
 - [RESTful API quick guide](https://www.tutorialspoint.com/restful/restful_quick_guide.htm)
    - Go through the Introduction, Resources, Methods, Statelessness and question-answer part of the tutorial.
  
-### Lesson
+### Lesson[(from bbaopen4u)](https://bbvaopen4u.com/en/actualidad/rest-api-what-it-and-what-are-its-advantages-project-development)
 
-To give a simple definition, REST is any interface between systems using HTTP to obtain data and generate operations on those data in all possible formats, such as XML and JSON. This is an increasingly popular alternative to other standard data exchange protocols such as SOAP (Simple Object Access Protocol), which have a high capacity but are also very complex. Sometimes it's preferable to use a simpler data-processing solution such as REST. 
+    To  give  a  simple  definition,  REST  is  any  interface  between  systems  using  HTTP  to  obtain  data  and  generate  operations  on  those data in  all  possible  formats,  such  as  XML  and  JSON.  This  is  an  increasingly  popular  alternative  to  other  standard  data  exchange protocols such as SOAP (Simple Object Access Protocol), which have a high capacity but are also very complex. Sometimes it's preferable to use a simpler  data-processing  solution  such  as  REST.
```

Comment:
All of your changes are on code notation in this file.  please change it to regular text.

Reply:
I did so as it was mention in the link that you shared. Do write in this way. Still, if you say I will change it

Reply:
OH, no I just meant please add the citation reference at the end the same way.  sorry.

Reply:
Ohk I will change that again.